src/gui/categoryfilterwidget.h

```diff
@@ -26,7 +26,23 @@
  * exception statement from your version.
  */
 
- #include <QTreeView>
+#include <QSortFilterProxyModel>
+#include <QTreeView>
+
+class CategoryFilterProxyModel: public QSortFilterProxyModel
+{
+    Q_OBJECT
```

Comment:
You don't need Q_OBJECT macro here since this class doesn't implement signals/slots (or any other metaobject features). And therefore, you have to move its declaration into .cpp.